docs/docs/rest-api/public/api/v2/types/volumeMount.raml

```diff
@@ -0,0 +1,62 @@
+#%RAML 1.0 DataType
```

Comment:
this type is very strange. i think too much got mushed together here. containers need to share access to the same volume. the specs for the volume should be defined ONCE at the pod level. containers can then choose how each volume is mounted into their own private mountns.